### PR TITLE
Resolves #73 #43 Home Page and navbar

### LINKED ISSUE #43: Navbar
- use same navbar as website
- Logo to the left
- Design, Components, About Tabs
- Tabs should collapse to a hamburger menu
   - make sure Sidebar gets included with hamburger menu if small

### LINKED ISSUE #73: Documentation Website Front Page
image

### PR DESCRIPTION
**Pull request by tko22**
image

Resolves #73 
Resolves #43 

is responsive as well

### CODE REVIEW

**Comment by SirajChokshi** [comment]
@tko22 The footer overflows to the right on Firefox 75 (and likely other browsers). For elements that use padding to space child elements, apply `box-sizing: border-box` so height/width account for padding.

Here is an image of the issue:

Screen Shot 2020-05-22 at 3 58 35 PM

File: docs/components/home/navbar.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+const NavBar = () => {
+  const router = useRouter();
+  const [showNav, toggleNav] = useState(false);
+
+  const onToggle = () => {
+    toggleNav(!showNav);
+  };
+
+  const [windowDimensions, setWindowDimensions] = useState({ width: 1000, height: 1000 });
```
Comment (jackieo5023) [inline docs/components/home/navbar.js:16]: Could this instead use the `useWindowDimensions` hook written?

Comment (tko22) [reply docs/components/home/navbar.js:16]: oh yeah I just removed `useWindowDimensions` because it doesn't work since its nextjs and server rendering

**Comment by tko22** [comment]
thx siraj

**Pull request merged**